Modules/Core/Common/include/itkHexahedronCell.hxx

```diff
@@ -366,14 +356,14 @@ HexahedronCell<TCellInterface>::EvaluatePosition(CoordRepType *            x,
     this->InterpolationDerivs(pcoords, derivs);
 
     //  calculate newton functions
-    for (unsigned int i = 0; i < Self::PointDimension3D; ++i)
-    {
-      fcol[i] = rcol[i] = scol[i] = tcol[i] = 0.0;
-    }
+    double fcol[Self::PointDimension3D] = { 0.0 };
+    double rcol[Self::PointDimension3D] = { 0.0 };
+    double scol[Self::PointDimension3D] = { 0.0 };
+    double tcol[Self::PointDimension3D] = { 0.0 };
```

Comment:
Nowadays, I tend to just write `{}` (rather than ` = { 0.0 }`, but that's just a matter of taste:

    double tcol[Self::PointDimension3D]{};